coruscant/src/reducers/projects.js

```diff
@@ -1,13 +1,58 @@
 import { normalize } from 'normalizr'
 import * as Constants from './../constants'
 import projectList from './../models/projectList'
+import issueList from './../models/issueList'
+import buildStatusList from './../models/buildStatusList'
 
 function makeInitialState() {
   return {
     all: {},
     owned: {},
     contributed: {},
-    fetchedUserProjects: false
+    issues: {},
```

Comment:
Are these the issues and build statuses for 1 project or all projects? This projects reducer object contains all the projects. So having the issues and build statuses here is really confusing. It would be better if this is moved to an issuesReducer and also a status reducer for better separation of concerns. For the status reducer for example, you would pull all the possible statuses from the backend and have it stored in the object. Then you can map it to each project using the id, similar to how a database would do it. For issues, you can fetch all the issues, map it to its respective project using the issue id - the project should contain a list of issue ids or each issue can contain a project id. Take a look at this to get a better idea of what I'm talking about - https://redux.js.org/recipes/structuring-reducers/normalizing-state-shape